File: app/src/main/java/com/grable/overflowingstack/models/OSResponse.java
package com.grable.overflowingstack.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by timgrable on 10/31/17.
 */

public class OSResponse {

    @SerializedName("items")
    @Expose
    private List<Question> items = null;

    @SerializedName("quota_remaining")
    @Expose
    private Integer quotaRemaining;

    public List<Question> getItems() {
        return items;
    }

    public void setItems(List<Question> items) {
        this.items = items;
    }

    public Integer getQuotaRemaining() {
        return quotaRemaining;
    }

    public void setQuotaRemaining(Integer quotaRemaining) {
        this.quotaRemaining = quotaRemaining;
    }
}

File: app/src/main/java/com/grable/overflowingstack/vone/models/OSResponse.kt
package com.grable.overflowingstack.vone.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.grable.overflowingstack.vtwo.models.Question

/**
 * Created by timgrable on 10/31/17.
 */

class OSResponse {
    @SerializedName("items")
    @Expose
    var items: List<Question> = arrayListOf()

    @SerializedName("quota_remaining")
    @Expose
    var quotaRemaining: Int? = null
}
